impl/doc/display.md

```diff
@@ -11,7 +11,7 @@ the supplied format, or an automatically inferred one.
 You specify the format on each variant by writing e.g. `#[display("my val: {}", some_val * 2)]`.
 For enums, you can either specify it on each variant, or on the enum as a whole.
 
-For variants that don't have a format specified, it will simply defer to the format of the
+For variants that don't have a format specified, it will simply delegate to the format of the
 inner variable. If there is no such variable, or there is more than 1, an error is generated.
```

Comment:
I think it would be good to explicitely document this similar behaviour for Debug too. Maybe even with an example where not all fields have a custom formatting to make it extra clear.

Reply:
@JelteF already done here:
https://github.com/JelteF/derive_more/pull/322/files#diff-abe7cf81ac108e356174682bd8788a7805b865bb066505bbb0e45efec2905ee5R87-R120